### PR TITLE
Move Hypothesis controls in articles

### PR DESCRIPTION
**Pull request by NuclearRedeye**

### CODE REVIEW

**Comment by discodavey** [comment]
There are only three pages that can be tested which are:
Position by first section - Default
Position centrally inline - inside eLife
Position end - digest

There is one more which position at the end of the second section which does not have page that is relevant to the code so this will need to be added to the list of things needed to be added to the pattern library.

@NuclearRedeye Can you please check you are happy with this and then we can revert the Hypothesis js file.

**Comment by discodavey** [comment]
I cannot seem to select you for reviewing @NuclearRedeye

File: source/_patterns/04-pages/article--research.mustache [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="global-wrapper">
+<div class="global-wrapper"  data-item-type="default">
```
Comment (NuclearRedeye) [inline source/_patterns/04-pages/article--research.mustache:1]: I know they result in the same output, but to match whats on prod it should be 'research-article'.

**Review by NuclearRedeye — COMMENTED** [review]
So some further changes are needed...
1. I thought on line 25 of `article--research.mustache` the class `content-container` needs to be removed right? Or is that no longer the case?
2. What about adding the `data-item-type` to the other templates?

**Comment by NuclearRedeye** [comment]
@discodavey That's because I'm the author of the PR. Can't be assigned as a reviewer to my own code! See my comments above, I think a few little changes are needed.

**Comment by discodavey** [comment]
@NuclearRedeye The other data types are in the js but we only have three pages set up that can take the annotations data type. I do not think we need to put it in every version of article research. There are other pages which are not in the pages section of the pattern library and these can be done as another ticket as I do not feel this is something that should be done now and can be done as part of pattern library update.

File: source/_patterns/04-pages/article--research.mustache (hunk 1 above)
Comment (discodavey) [reply source/_patterns/04-pages/article--research.mustache]: I have changed the one for the article section.

**Pull request closed without merging**